sdks/typescript/src/session.ts

```
@@ -1,151 +1,194 @@
+    peers:
+      | string
+      | Peer
+      | Array<string | Peer>
+      | [string | Peer, SessionPeerConfig]
+      | Array<[string | Peer, SessionPeerConfig]>
+      | Array<string | Peer | [string | Peer, SessionPeerConfig]>
+  ): Promise<void> {
+    const peerDict: Record<string, SessionPeerConfig> = {}
     if (!Array.isArray(peers)) {
-      peers = [peers];
+      peers = [peers]
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)